merge-release-run.js

```diff
@@ -50,7 +50,10 @@ const run = async () => {
   process.stdout.write(execSync(`npm version --allow-same-version=true --git-tag-version=false ${current} `))
   let newVersion = execSync(`npm version --git-tag-version=false ${version}`).toString()
   console.log(newVersion)
+  process.stdout.write(execSync(`git commit -a --amend --no-edit`))
   process.stdout.write(execSync(`npm publish --access=public`))
-  process.stdout.write(execSync(`git checkout package.json`))
+  process.stdout.write(execSync(`git push`))
```

Comment:
Pushing so the npm package and github repo have the same commits/hashes

Reply:
This creates a loop condition where the publish itself triggers additional pushes which trigger more publishes. This is fine if you only have one push action because GitHub won’t trigger a push action caused by that action, but if you have two this could get tricky. This is why I’ve taken to just putting `0.0.0-dev` as the master version and leaving it be.

Reply:
I thought the same thing however given the code on [line 29](https://github.com/mikeal/merge-release/blob/2a12a1679c2dc893e76223d511b22bab6ad2f73d/merge-release-run.js#L29) it should prevent it.

The git push happens after the publish which means the hash of the latest version on npm will be the same as the hash of the second trigger making it return without triggering forever unless I am missing something.

Reply:
hrm... ok. i think i like this change, but we should bump the major version since we’re now pushing things into the commit log that we weren’t before.